tests/recovery/recoverySecurity.test.ts

```
@@ -371,5 +371,81 @@ describe('Recovery Security E2E Tests', () => {
+
+      const unlockResult = await unlockApp({
+        identifier: 'test@example.com',
+        passcode: 'test1234',
+        deviceFingerprint: attackerDevice,
+      });
+
+      // Fetch the real code hash so we can construct the correct OTP value
+      // In this test we instead force the row into a locked state directly
+      const otpRow = await prisma.otpChallenge.findFirst({
+        where: { userId: testUser.id, usedAt: null, lockedAt: null },
+        orderBy: { createdAt: 'desc' },
```

ESLint

[error] Replace `'desc'` with `"desc"`

(prettier/prettier)